Aztec/Classes/Formatters/Implementations/MarkFormatter.swift

```diff
@@ -4,28 +4,43 @@ import UIKit
 class MarkFormatter: AttributeFormatter {
 
     var placeholderAttributes: [NSAttributedString.Key: Any]?
+    var textColor: String?
+    var defaulTextColor: UIColor?
 
     func applicationRange(for range: NSRange, in text: NSAttributedString) -> NSRange {
         return range
     }
 
     func apply(to attributes: [NSAttributedString.Key: Any], andStore representation: HTMLRepresentation?) -> [NSAttributedString.Key: Any] {
-       var resultingAttributes = attributes
+        var resultingAttributes = attributes
+        let colorStyle = CSSAttribute(name: "color", value: textColor)
+        let backgroundColorStyle = CSSAttribute(name: "background-color", value: "rgba(0, 0, 0, 0)")
+
+        let styleAttribute = Attribute(type: .style, value: .inlineCss([backgroundColorStyle, colorStyle]))
+        let classAttribute = Attribute(type: .class, value: .string("has-inline-color"))
 
-        var representationToUse = HTMLRepresentation(for: .element(HTMLElementRepresentation.init(name: "mark", attributes: [])))
+        // Setting the HTML representation
+        var representationToUse = HTMLRepresentation(for: .element(HTMLElementRepresentation.init(name: "mark", attributes: [styleAttribute, classAttribute])))
         if let requestedRepresentation = representation {
             representationToUse = requestedRepresentation
         }
         resultingAttributes[.markHtmlRepresentation] = representationToUse
 
+        if textColor != nil {
```

Comment:
Optional nit: I'd consider an `if let` here to unwrap `textColor` safely so we don't have to force unwrap `textColor` later. It should be safe as is, just a preferred idiom.

Reply:
Thanks for the suggestion! Addressed in https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1352/commits/28e62ab8f254eec55b87e4a5607e2c9d4a9887d4